app/models/user.rb

```diff
@@ -115,6 +115,10 @@ def student?(course)
     course.users.role('student').include? self
   end
 
+  def isstudent?
+    courses.any? { |course| student?(course) }
+  end
+
```

Comment:
This method and the `isinstructor?` method are going to be easy to confuse with the prior `student?` and `instructor?` methods.

Perhaps instead, `student_role?` and `instructor_role?`. Or maybe `course_student?` and `course_instructor?`.